### ISSUE TITLE
Click on DOM will cause it trigger a mouseleave/mouseout event when used in electron.

### ISSUE THREAD

**Issue by zerotung** [wilix-team/iohook#141]
## Expected Behavior
This should behave the same as in the browser, Clicking on DOM should not let it trigger a mouseout/mouseleave event.

## Current Behavior
DOM will trigger a mouseleave/mouseout event when click on it.

## Steps to Reproduce (for bugs)
```javascript
// main.js
const iohook = require('iohook');
iohook.start();
```
```html
<!DOCTYPE html>
<html>

<head>
  <meta charset="UTF-8">
  <title>Hello World!</title>
  <style>
    .iohook {
      width: 100px; height: 100px; background-color: #ccc;
    }
    .iohook:hover {
      background-color: #c00;
    }
  </style>
</head>

<body>
    <div class="iohook"></div>
  </div>
  <script>
    var $iohook = document.getElementsByClassName('iohook')[0];
    $iohook.addEventListener('mouseenter', function() { console.log('mouseenter'); });
    $iohook.addEventListener('mouseout', function() { console.log('mouseout'); });
    $iohook.addEventListener('mouseover', function() { console.log('mouseover'); });
    $iohook.addEventListener('mouseleave', function() { console.log('mouseleave'); });
  </script>
</body>

</html>
```
1. Move the mouse into the box where the class is iohook, will log `mouseover`, `mouseenter`
2. Click on it, will log `mouseout`, `mouseleave`
3. Move the mouse around a little bit in the box, will log `mouseover`, `mouseenter`
4. Move the mouse out of box, will log `mouseout`, `mouseleave`

## Context
In my project, some buttons appear only if their parent node is in hover state. This issue will cause buttons disappear when click on them, and appear again if move arround a little bit.

## Your Environment
* Version used: `iohook@0.3.1`
* Environment name and version (e.g. Chrome 39, node.js 5.4): node.js `v10.5.0`, electron `v2.0.9`
* Operating System and version (desktop or mobile): `Windows 10 x64`

**Comment by Djiit** [comment]
Hi there,

I don't get where are you using iohook here ? your main.js isn't included in your HTML file.

**Comment by zerotung** [comment]
> Hi there,
> 
> I don't get where are you using iohook here ? your main.js isn't included in your HTML file.

Thanks a lot for reply!

I just build a [demo](https://github.com/zerotung/electron-iohook-demo) for this issue refer to electron-quick-start template. 

By the way, I tested it on my MacBook, and it won't be reproduced on my MacBook. I think this is a bug that only appears on Windows operating systems.

**Comment by zerotung** [comment]
Yes, I did not call any iohook interfaces in the demo. I just want to explain that, as long as I import iohook lib and run `iohook.start()`, it causes problems.

Normally in web page, it doesn't seem to trigger mouseout/mouseleave event on click on DOM element.

**Comment by ash0x0** [comment]
Is this still an issue with recent versions?
